src/emu/rendersw.hxx

```diff
@@ -128,10 +128,10 @@ private:
 	//  palettized 16bpp source
 	//-------------------------------------------------
 
-	static inline u32 get_texel_palette16(const render_texinfo &texture, s32 curu, s32 curv)
+	static constexpr u32 get_texel_palette16(const render_texinfo &texture, s32 curu, s32 curv)
```

Comment:
Is marking this `constexpr` portable given it uses `rgbaint_t` which uses various vector intrinsics?